ansible/roles_ocp_workloads/ocp4_workload_dm7_foundations_elt/tasks/post_workload.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Implement your Post Workload deployment tasks here
+# --------------------------------------------------
+
+- name: Get console route
+  k8s_info:
+    kind: Route
+    api_version: route.openshift.io/v1
+    namespace: openshift-console
+    name: console
+  register: r_console_route
+
+- name: Set webconsole address
+  set_fact:
+    webconsole: "https://{{ r_console_route.resources[0].spec.host }}"
+
+- name: Get API for command line
+  command: oc whoami --show-server
```

Comment:
We prefer this code now - no shell out.

  - name: Determine API server URL
    k8s_info:
      api_version: config.openshift.io/v1
      kind: Infrastructure
      name: cluster
    register: r_api_url